articles/applications-of-artificial-intelligence-(ai)-in-design-systems/index.md

```diff
@@ -0,0 +1,72 @@
+- Design Characteristics – It is the stage for defining each system element's architecture and characteristics; also, the definition of the viability.
+- Alternatives for obtaining system elements
+- Managing the design
+
+### [Artificial Intelligence (AI)](https://www.section.io/engineering-education/artificial-intelligence-future/)
+
+It is human intelligence in machines that are programmed to think like a machine like a human and their actions. A computer uses machine learning to determine how they respond to these situations.
+
+### Real applications of AI in design systems
+
+#### Website development
+
+The use of tools like Tobii Pro Sprint, web-based eye-tracking technology, is used in developing websites. It allows UX designers to record website users' eye movements, thereby assisting in improving website design.
+
+The grid is the first company to offer the website design within minutes. The websites are custom AI-designed. After The Grid, many other companies have emerged with the same AI-designed websites, such as [Wix](https://www.wix.com/), and [Squarespace](https://www.squarespace.com/).
+
+With AI technology, one only needs to upload the data content; and build the website from the ground within minutes.
+
+The same technology is used in design logos by the [Turbologo](https://turbologo.com/?ref=hackernoon.com). One need only upload the brand name, colors, and other preferences, and the logo is generated within minutes.
+
+#### Automation of legwork
+
+Technology can easily automate most of the designer tasks. Using AI, tasks like cropping images and correcting colors can be easily be automated. [Adobe Sensei](https://www.adobe.com/experience-platform/intelligent-services.html?ref=hackernoon.com) is a design tool that uses intelligent features. It assists the designers to perform work more efficiently and faster. Adobe Sensei identifies patterns in the image, edit, or even creates a new scene.
+
+Another example is Netflix, which uses [augmented intelligence](https://www.section.io/engineering-education/introduction-to-extended-reality/) system for artwork personalization. When the need arises to create multiple banners, designers only need to look at the robot's banners to accept or reject them.
+
+#### DesignScape
+
+It is a system that helps in the design process by aiding in suggestions such as changes in position, scale, and elements.
+
+It assists the website designers with suggestions to create near-perfect websites. It doesn't design the website on its own, though it improves the designer's efficiency. Using it reduces maintenance efforts and time taken to complete the task.
+
+#### [Font Joy](https://fontjoy.com/)
+
+It uses machine learning to generate font combinations. Assists designers in choosing the best font combination. A person needs just to set different fonts; once done, click generate button. The page will load and will generate a new font combination.
+
+### Benefits of using AI in design systems
```

Comment:
The title says Applications of AI in Design systems. What systems are we talking about here? We need to be more specific. I don't think these qualify as systems. They are software. @hectorkambow @Pius1991 What do you think about this?

Reply:
> The title says Applications of AI in Design systems. What systems are we talking about here? We need to be more specific. I don't think these qualify as systems. They are software. @hectorkambow @Pius1991 What do you think about this?

@lalith1403 I think "Applications of AI in Design Technologies" would sound better.